app/services/EventReportService.scala

```diff
@@ -450,17 +452,23 @@ class EventReportService @Inject()(eventReportConnector: EventReportConnector,
       }
     }
 
-    for {
-      transformed1828Payload <- Future.fromTry(toTry(userAnswersJson.transform(API1828.transformToETMPData)))
-      _ <- recoverAndValidatePayload(transformed1828Payload)
-    } yield {
-      ()
+    declarationLockRepository.insertLockData(pstr, psaPspId).flatMap { isAvailable => //TODO: Rename refer to AFT
```

Comment:
Should this TODO comment be removed. I think it is names appropriately.

Reply:
removed todo comments and renamed method